docs/chapter1/intermediate/app-archetype/env-vars.md

```diff
@@ -27,15 +26,6 @@ Some of the app archetype's features can be controlled by environment variables.
 | `ELECTRODE_LOAD_DLLS`          |                                                                                                                                                                                                                                                                  If non empty json, each entry is the name of Electrode Webpack DLL to be loaded by the app                                                                                                                                                                                                                                                                  |          `{}` |
 | `WEBPACK_MINIFY`               |                                                                                                                                                                                                                                                                                   If `true`, webpack in `development` mode to minify code                                                                                                                                                                                                                                                                                    |        `true` |
 
-- The following settings will apply only if `WEBPACK_DEV_MIDDLEWARE` is not `true`, meaning a separate webpack-dev-server is used for development:
-
-| Variable                    |                        Description                         |     Default |
-| --------------------------- | :-------------------------------------------------------- | :----------: |
-| `WEBPACK_DEV_HOST`          |  If defined, used as the hostname for webpack dev server   | `localhost` |
-| `WEBPACK_DEV_PORT`          | If defined, used as the port number for webpack dev server |      `2992` |
```

Comment:
our internal webpack dev server still look at `WEBPACK_DEV_HOST` and `WEBPACK_DEV_PORT`, may want to work them into the doc.